rpm/podman.spec

```diff
@@ -245,6 +237,10 @@ LDFLAGS="-X %{ld_libpod}/define.buildInfo=${SOURCE_DATE_EPOCH:-$(date +%s)} \
          -X %{ld_libpod}/config._etcDir=%{_sysconfdir} \
          -X %{ld_project}/pkg/systemd/quadlet._binDir=%{_bindir}"
 
+%if %{defined copr_build}
+LDFLAGS="$LDFLAGS -X %{ld_libpod}/define.gitCommit=##GIT_COMMIT##"
```

Comment:
if for whatever reason the sed doesn't happen/work we write "##GIT_COMMIT##" to the commit which does not like good.
Maybe define `GIT_COMMIT=` on the line above and them use `-X %{ld_libpod}/define.gitCommit=$GIT_COMMIT` sed the GIT_COMMIT line to add the sha. In that case if it fails it stays empty.

Also it would be good to add a comment that thi sis supposed to be replaced before the build via `.packit.sh` so a normal person just reading the spec understands the connection.

Reply:
added, setting back to Draft until I verify change.

Reply:
Done.